### PR TITLE
Add blacklist functionality to party list app

### PR DESCRIPTION
**Pull request by austintrose**
Solved merge conflicts. 

New endpoint causing weirdness, still working on that.

### CODE REVIEW

**Comment by tmwbook** [comment]
Just went through and resolved conflicts.  Still have to do testing.  Basically merged the last minute changes before paint party #testonprod #weredoinitlive

**Comment by tmwbook** [comment]
So the feature works, but there are a couple of things that I'll add.
1)  We can't remove people from the blacklist without going in as admin.  I think this should be a feature of the first councilor permissions
2) The list will warn you when you add someone, but it doesn't prompt you for confirmation that you would still like to add the person.

**Comment by tmwbook** [comment]
All the features are functioning now.  Debating styling the radio buttons.  I've tested the feature as admin and as a lowest permission user

File: SigmaPi/PartyList/templates/blacklist.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{% extends "party_base.html" %}
```
Comment (co1in) [inline SigmaPi/PartyList/templates/blacklist.html]: Need to add validation / error message for empty name and/or description

**Comment by kdmccormick** [comment]
Merges with master are weird. Can you rebase on to master instead?

**Comment by kdmccormick** [comment]
Closed in favor of #52

**Pull request closed without merging**